### ISSUE TITLE
Cannot read property 'backgroundColor' of undefined

### ISSUE THREAD

**Issue by mortenbirkelund** [danielbayerlein/dashboard#122]
Cloning this repository, and running install, build and start command, I get the following error as soon as I visit the dashboard with my browser. 

```
(pi@HulenPi:~/dashboard/pages $ npm start

> dashboard@1.0.0 start /home/pi/dashboard
> next start

> Ready on http://localhost:3000
TypeError: Cannot read property 'backgroundColor' of undefined
    at /home/pi/dashboard/.next/server/static/mX9Kv1iErUfmWQOdMcvjO/pages/index.js:363:30
    at flatten (/home/pi/dashboard/node_modules/styled-components/dist/styled-components.cjs.js:1292:21)
    at flatten (/home/pi/dashboard/node_modules/styled-components/dist/styled-components.cjs.js:1272:16)
    at ComponentStyle.generateAndInjectStyles (/home/pi/dashboard/node_modules/styled-components/dist/styled-components.cjs.js:1481:19)
    at StyledComponent.generateAndInjectStyles (/home/pi/dashboard/node_modules/styled-components/dist/styled-components.cjs.js:2160:36)
    at StyledComponent.renderInner (/home/pi/dashboard/node_modules/styled-components/dist/styled-components.cjs.js:2065:33)
    at a.render (/home/pi/dashboard/node_modules/react-dom/cjs/react-dom-server.node.production.min.js:44:64)
    at a.read (/home/pi/dashboard/node_modules/react-dom/cjs/react-dom-server.node.production.min.js:41:58)
    at renderToString (/home/pi/dashboard/node_modules/react-dom/cjs/react-dom-server.node.production.min.js:53:83)
    at render (/home/pi/dashboard/node_modules/next-server/dist/server/render.js:86:16))
```

When running "npm run dev" then everything works fine.

**Comment by danielbayerlein** [comment]
Thanks for the report. I'll fix the problem ASAP.

**Comment by danielbayerlein** [comment]
@mortenbirkelund a33a953d4a07087c9813908fdda6271076dd3fc2 fixes the problem temporarily.

**Comment by mortenbirkelund** [comment]
Just tested, and I still get the same error even with a33a953

**Comment by danielbayerlein** [comment]
@mortenbirkelund Thank you for your feedback. I'll investigate the problem.

**Comment by mortenbirkelund** [comment]
Could be an issue that is related to my setup. I have tried some different earlier versions and i am getting the same error. Any suggestion as what i should look into?

**Comment by danielbayerlein** [comment]
I can reproduce the problem with Next.js v8.1.0

**Comment by marco-streng** [comment]
Seems to be fixed in Next.js `v8.1.1-canary.40`

**Comment by yatong-moto** [comment]
I tried the latest version "9.0.1", it fixes the problem too.

**Comment by danielbayerlein** [comment]
@yatong-moto Thank you for the feedback! Fixed with 084dfdb237db02f37b17c5c060be29a794729a76
